src/test/kotlin/com/karumi/katagenda/ui/ContactsListPresenterTest.kt

```diff
@@ -0,0 +1,163 @@
+package com.karumi.katagenda.ui
+
+import com.karumi.katagenda.domain.Contact
+import com.karumi.katagenda.usecase.AddContact
+import com.karumi.katagenda.usecase.GetContacts
+import com.nhaarman.mockito_kotlin.argumentCaptor
+import com.nhaarman.mockito_kotlin.times
+import com.nhaarman.mockito_kotlin.verify
+import com.nhaarman.mockito_kotlin.whenever
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.MockitoAnnotations
+
+class ContactsListPresenterTest {
+
+    companion object {
+
+        private val ANY_NUMBER_OF_CONTACTS = 7
+        private val ANY_FIRST_NAME = "Pedro Vicente"
+        private val ANY_LAST_NAME = "Gomez Sanchez"
+        private val ANY_PHONE_NUMBER = "666666666"
+    }
+
+    @Mock private val view: ContactsListPresenter.View? = null
```

Comment:
Can we configure them as `lateinit` vars instead? We'd be able to stop using optionals as long as we trust our test to create the instances before using them.

Reply:
This is how to define it, the good thing is that it won't be optional (and we get rid of the multiple `?`, `!!` symbols), the bad side is that it has to be a variable but we won't use it as such

```kotlin
@Mock private lateinit var view: ContactsListPresenter.View
```